docs/tutorial_1/1_beam_introduction.ipynb

```diff
@@ -1,876 +1,1681 @@
+    "- Ratings are on a five star (integral) scale from 1 to 5.\n",
+    "- Dates have the format YYYY-MM-DD.\n",
+    "```\n",
+    "\n",
+    "A more thorough description of each of those files is available in the [`README`](netflix_prize_dataset/data/README) file. "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### Preparing our dataset \n",
+    "\n",
+    "The size of the whole dataset ($\\approxeq$ `2GB` if joining together all the `combined_data_` files) doesn't allow to process the entire collection as quickly and interactive as we would expect into an interactive environment like Jupyter notebooks. This is still true despite all the boosting trick we will be setting up on `beam` later, nonetheless. \n",
+    "\n",
+    "For this reason, we will limit our analysis to a reduced version of the dataset. In the next two cells, we will quickly gather some statistics on the size of each data file (in terms of `number of lines`), and we will interactively choose how many of these lines we will retain in our working version.\n",
+    "\n",
+    "**Note**: Without any loss of generality, we will use `combined_data_1.txt` as for the reference data file. As you will soon realise, it won't make much difference. However, feel free to experiment by changing the reference file, or even by joining all of them together, if you feel your computing environment is able to handle this dataset in a reasonable amount of time (_more on this later_)."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "#@title Let's see how many lines each data file contains { display-mode: \"form\" }\n",
+    "data_files = filter(lambda f: f.startswith(\"combined_data_\"), os.listdir(DATASET_FOLDER))\n",
+    "lines_count = 0\n",
+    "for data_file in sorted(data_files):\n",
+    "    file_path = DATASET_FOLDER / data_file\n",
+    "    wc_l = !wc -l $file_path\n",
+    "    lines, _ = wc_l[0].strip().split()\n",
+    "    lines_count += int(lines)\n",
+    "    print(f\"{data_file}:  {int(lines):,}\")\n",
+    "print(f\"Total Lines Count  : {lines_count:,}\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "From these statistics, we can immediately notice that each data file roughly contains the same number of lines, which means that the amout of time required to process each one of them will be more or less the same.\n",
+    "\n",
+    "However, even just limiting our analysis to one single file (i.e. `>20M` lines) might be prohibitive, and indeed beyond of the scope of this tutorial. \n",
+    "\n",
+    "The **main goal** of the tutorial is to demostrate the ability of `apache_beam` to scale up the computation on large datasets. For real scalability however, we might also need decent computation capacity (_and we generally don't, if running this notebook on our laptop_).\n",
```

Comment:
It's hard to demonstrate how to scale up computations on 1 machine :). I'd recommend to remove this sentence

Reply:
Yep, you're right. Rephrased the paragraph with: 

```
However, even just limiting our analysis to one single file (i.e. >20M lines) might be prohibitive, and indeed beyond of the scope of this tutorial.
For real scalability experiments, we would need better computation capacity (which generally goes by far beyond laptops specs).
```